### DISCUSSION TITLE
Flutter Web + Tauri Template

### DISCUSSION THREAD

**Discussion by rodydavis** [tauri-apps/tauri#6328] [Ideas]
## Motivation

- Gain access to the flutter/dart package ecosystem
- Flutter interop with Rust
- Leverage NPM ecosystem
- Render app with Flutter (or embed to just part of the page)

## Example

https://github.com/rodydavis/flutter_tauri

**Comment by leedstyh** [comment]
Hi @rodydavis, any idea how to open multi window in dart code.

**Reply by rodydavis** [reply]
Oh interesting, probably would do it in rust and just call the function via dart

**Reply by leedstyh** [reply]
Got you, thank you so much.
